### PR TITLE
Edit open hardware

### LINKED ISSUE #3371: [Image needs adding] Open Hardware subchapter
### Summary

The chapter on open hardware: https://the-turing-way.netlify.app/reproducible-research/open/open-hardware.html does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

### What needs to be done?

- [ ] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [ ] Write the alt text for the image
- [ ] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by kallewesterling**
Adding image and alt-text for the Open Hardware section. Resolves #3371.

### CODE REVIEW

**Comment by kallewesterling** [comment]
For reference, #3331.

File: book/website/reproducible-research/open/open-hardware.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ It is worth noting that open hardware does not necessarily mean free.
 Units may still be sold (by the original designer or by others), but users *could* build them from scratch.
 Whether or not they choose to buy the unit, users can still get a full understanding of how the hardware works from open documentation, designs, and similar.  
 
+```{figure} ../../figures/open-hardware.*
+---
+height: 400px
+name: open-hardware
+alt: Cartoon-like sketch of the open hardware workflow. At the top, a phrase reads "Cultivated ecosystem of open knowledge," connected to an illustration of a garden representing the open hardware concept. The process flows from the first steps of articulating a need and starting to prototype, into steps like "selection/demo," followed by "documentation," and culminating in "publishing." Three frames below the garden metaphorically illustrate the open hardware's reuse, replication, and reproduction. The bottom of the image features people in various acts of sharing and reaching for seeds, floating down from the published documentation, representing a communal and iterative nature of open hardware development.
```
Comment (aleesteele) [inline book/website/reproducible-research/open/open-hardware.md:19]:
```suggestion
alt: Cartoon-like sketch of the open hardware workflow using garden imagery and tools, connected by arrows in a cycle. The first steps are articulating a need and starting to prototype, featuring a person thinking about a seedling, and a few different visual sketches of what a flower could look like. This flows into steps like "selection/demo," that features a person presenting the sketch and tools. This is followed by "documentation", featuring people in various acts of sharing and reaching for seeds, representing the communal and iterative nature of open hardware development. This process culminates in "publishing", illustrated by a person releasing seedlings out of a window. This points to three frames that metaphorically illustrate the open hardware's reuse, replication, and reproduction of through the creation of connected seedlings. The final step is labeled "Cultivated ecosystem of open knowledge," connected to an illustration of a garden of flowers that is being water. 
```

**Review by Arielle-Bennett — APPROVED** [review]

File: book/website/reproducible-research/open/open-hardware.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kallewesterling) [reply book/website/reproducible-research/open/open-hardware.md]: Fixed it with https://github.com/the-turing-way/the-turing-way/pull/3374/commits/f27e7e4b278a35cbafa9579a4179891afc017029, thanks!

**Comment by kallewesterling** [comment]
Let me know if you have any more feedback, @Arielle-Bennett! :)

**Review by Arielle-Bennett — APPROVED** [review]
No more feedback from me, please go ahead and merge!

**Pull request merged**